library/vyos_facts.py

```diff
@@ -29,15 +27,16 @@
     respective resource name.  The facts module will always collect a
     base set of facts from the device and can enable or disable
     collection of additional facts.
-author: Nilashish Chakraborty (@nilashishc)
+author: [u'Nilashish Chakraborty (@NilashishC)', u'Rohit Thakur (@rohitthakur2590)']
 options:
   gather_subset:
     description:
       - When supplied, this argument will restrict the facts collected
         to a given subset. Possible values for this argument include
-        all, min, neighbor and config. Can specify a list of values to
-        include a larger subset. Values can also be used with an initial
-        C(M(!)) to specify that a specific subset should not be collected.
+        all, min, hardware, config, legacy, and interfaces. Can specify a
```

Comment:
Please have a look at vyos_facts gather_subset description in ansible/ansible.
```suggestion
        all, default, config, and neighbors. Can specify a
```